pkgs/tools/X11/sct/default.nix

```diff
@@ -1,31 +1,26 @@
-{stdenv, fetchurl, libX11, libXrandr}:
+{ stdenv, fetchurl, libX11, libXrandr }:
+
 stdenv.mkDerivation rec {
   name = "sct";
-  buildInputs = [libX11 libXrandr];
+
   src = fetchurl {
     url = http://www.tedunangst.com/flak/files/sct.c;
     sha256 = "01f3ndx3s6d2qh2xmbpmhd4962dyh8yp95l87xwrs4plqdz6knhd";
-    
-    # Discussion regarding the checksum and the source code can be found in issue #17163 
-    # The code seems unmaintained, yet an unknown (probably small change) in the code caused 
-    # failed builds as the checksum had changed.
-    # The checksum is updated for now, however, this is unpractical and potentially unsafe 
-    # so any future changes might warrant a fork of the (feature complete) project. 
-    # The code is under public domain.
```

Comment:
can you keep this comment.

Reply:
Can we host this file somewhere else, where it won't be changed? Otherwise it will break nixpkgs.

Reply:
@Mic92 This was already decided in #17163. Author (one of OpenBSD commiters, tedu@) knows that several distros rely on this file being unchanged, and probably hasn't anticipated this when the file was changed previously. I could have made https://web.archive.org copy and link to it, but unfortunately author's website uses a custom TLS certificate and https://web.archive.org can't archive when certificate is not valid/in cert store.